### PR TITLE
Adding documentation about deploying Project to Dataverse instances. Cleaned up references to CDS

### PR DESCRIPTION
**Pull request by BeReasonable**
There is some new content related to deploying Project to Sandbox and Production Dataverse instances. There are also cleaned up references to CDS (which has been renamed Dataverse).

### CODE REVIEW

**Comment by BeReasonable** [comment]
@efrene , is this something that you can help with?

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: ProjectWeb/deploying-project.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+audience: admin
+ms.topic: article
+ms.service: project-web
+localization_priority: Normal
+description: "Learn how to deploy Project to the Dataverse."
+---
+
+## Deploying Project
+
+> [!Note] This section refers to upcoming functionality. Project can only be deployed to the default Dataverse instance. It is not yet possible to deploy Project to other types of Dataverse instances.
```
Comment (JohanFreelancer9) [inline ProjectWeb/deploying-project.md:16]:
```suggestion
> [!Note]
> This section refers to upcoming functionality. Project can only be deployed to the default Dataverse instance. It is not yet possible to deploy Project to other types of Dataverse instances.
```

File: ProjectWeb/deploying-project.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Project for the web is available for use in the Default Environment as well as in Sandbox and Production Dataverse environments.
+
+Project for the web in the Default environment enables customers to quickly get started creating projects, managing schedules, and sharing them with other users in the organization. Because everyone is a member of this environment by default, enabling users to create and manage Projects only requires that you assign a Project license to them.
+
+For some situations, you should consider deploying Project to additional environments. These are:
+* Application Lifecycle management (Development/Test/Production)
+* Data residency requirements for when the tenant is in a different geography than the users
+* Customizing Project to behave differently for different business units
+
+If you are looking to deploy to additional environments, you will need to create the environments, deploy project, and configure access for users.
```
Comment (JohanFreelancer9) [inline ProjectWeb/deploying-project.md:27]:
```suggestion
If you are looking to deploy to additional environments, you will need to create the environments, deploy Project, and configure access for users.
```

File: ProjectWeb/deploying-project.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+[Learn more about Dataverse environments]([https://docs.microsoft.com/power-platform/admin/environments-overview)
+
+## Provisioning a new environment
+> [!Note] This section only applies to Admins interesting in deploying Project to a non-Default environment.
```
Comment (JohanFreelancer9) [inline ProjectWeb/deploying-project.md:32]:
```suggestion
> [!Note]
> This section only applies to Admins interested in deploying Project to a non-Default environment.
```

File: ProjectWeb/deploying-project.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+Project is supported in the following types of environments
+* Default
+* Production
+* Sandbox
+
+To be able to deploy to Sandbox and Production environments, the environment needs to be created with a database and the "Enable Dynamics 365 Apps" toggle *must* be disabled.
+ ![D365 App Toggle](media/AppToggle.png) 
+
+[Learn more about creating and managing environments](https://docs.microsoft.com/en-us/power-platform/admin/create-environment)
```
Comment (JohanFreelancer9) [inline ProjectWeb/deploying-project.md:42]:
```suggestion
[Learn more about creating and managing environments](https://docs.microsoft.com/power-platform/admin/create-environment)
```

File: ProjectWeb/connect-to-project-for-the-web-data-through-powerbi-desktop.md [hunk 5]
```diff
@@ -113,7 +113,7 @@ If you are still not sure, check with your Office 365 admin and have them check
 After Power BI Desktop retrieves the data, the visualizations in each report page will load and display the data.  
 
 > [!Note]
```
Comment (JohanFreelancer9) [inline ProjectWeb/connect-to-project-for-the-web-data-through-powerbi-desktop.md:115]:
```suggestion
> [!NOTE]
```

File: ProjectWeb/deploying-project.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+Deployment of Project to the Default environment is done for you automatically. When Project for the web or Roadmap are first used in an Office 365 tenant, a Default Dataverse instance is provisioned for the tenant and the solutions are deployed.
+
+### Deploying to a non-Default environment
+Deploying Project to a non-Default environment is done from within the [Power Platform Admin Center (PPAC)](https://admin.powerplatform.microsoft.com). 
+
+Open the **Resources > Dynamics 365 apps** page from the left-hand navigation menu. Then, install the **Project Service Core** package into your environment.
+![Project package](media/InstallProject.png) 
+
+> [!Note] If the installation package isn't appearing in the list of     available packages, either the tenant doesn't have a Project license, or the environment was created with the "Enable Dynamics 365 Apps toggle" enabled.
```
Comment (JohanFreelancer9) [inline ProjectWeb/deploying-project.md:56]:
```suggestion
> [!NOTE]
> If the installation package isn't appearing in the list of available packages, either the tenant doesn't have a Project license, or the environment was created with the "Enable Dynamics 365 Apps toggle" enabled. 
```

File: ProjectWeb/deploying-project.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+### Deploying to a non-Default environment
+Deploying Project to a non-Default environment is done from within the [Power Platform Admin Center (PPAC)](https://admin.powerplatform.microsoft.com). 
+
+Open the **Resources > Dynamics 365 apps** page from the left-hand navigation menu. Then, install the **Project Service Core** package into your environment.
+![Project package](media/InstallProject.png) 
+
+> [!Note] If the installation package isn't appearing in the list of     available packages, either the tenant doesn't have a Project license, or the environment was created with the "Enable Dynamics 365 Apps toggle" enabled. 
+
+
+[Learn more about using the PPAC to deploy applications](https://docs.microsoft.com/en-us/power-platform/admin/manage-apps)
```
Comment (JohanFreelancer9) [inline ProjectWeb/deploying-project.md:59]:
```suggestion
[Learn more about using the PPAC to deploy applications](https://docs.microsoft.com/power-platform/admin/manage-apps)
```

File: ProjectWeb/deploying-project.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+Project for the web in the Default environment enables customers to quickly get started creating projects, managing schedules, and sharing them with other users in the organization. Because everyone is a member of this environment by default, enabling users to create and manage Projects only requires that you assign a Project license to them.
+
+For some situations, you should consider deploying Project to additional environments. These are:
+* Application Lifecycle management (Development/Test/Production)
+* Data residency requirements for when the tenant is in a different geography than the users
+* Customizing Project to behave differently for different business units
+
+If you are looking to deploy to additional environments, you will need to create the environments, deploy project, and configure access for users.
+
+[Learn more about Dataverse environments]([https://docs.microsoft.com/power-platform/admin/environments-overview)
```
Comment (JohanFreelancer9) [inline ProjectWeb/deploying-project.md:29]:
```suggestion
[Learn more about Dataverse environments](https://docs.microsoft.com/power-platform/admin/environments-overview)
```

File: ProjectWeb/deploying-project.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+> [!Note] If the installation package isn't appearing in the list of     available packages, either the tenant doesn't have a Project license, or the environment was created with the "Enable Dynamics 365 Apps toggle" enabled. 
+
+
+[Learn more about using the PPAC to deploy applications](https://docs.microsoft.com/en-us/power-platform/admin/manage-apps)
+
+## Configuring Roles and Security
+
+Sandbox and Production environments require additional configuration. Assign the **Project Common** and **Project User** roles to anyone who will be creating Projects in the environment. Also, ensure these users have the appropriate Project license.
+
+There is no additional configuration needed to enable users to manage projects in the default environment. Users in the Default Environment only need a Project license to be able to create and manage Projects.
```
Comment (JohanFreelancer9) [inline ProjectWeb/deploying-project.md:65]:
```suggestion
There is no additional configuration needed to enable users to manage Projects in the default environment. Users in the default environment only need a Project license to be able to create and manage Projects. 
```

File: ProjectWeb/deploying-project.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+## Configuring Roles and Security
+
+Sandbox and Production environments require additional configuration. Assign the **Project Common** and **Project User** roles to anyone who will be creating Projects in the environment. Also, ensure these users have the appropriate Project license.
+
+There is no additional configuration needed to enable users to manage projects in the default environment. Users in the Default Environment only need a Project license to be able to create and manage Projects. 
+
+[Learn more about security roles in the Dataverse](https://docs.microsoft.com/power-platform/admin/security-roles-privileges)
+
+> [!Note] Project-related roles are only available after the Project Service Core package has been deployed to the environment
```
Comment (JohanFreelancer9) [inline ProjectWeb/deploying-project.md:69]:
```suggestion
> [!NOTE]
> Project-related roles are only available after the Project Service Core package has been deployed to the environment.
```

File: ProjectWeb/export-user-data-from-project-for-the-web.md [hunk 11]
```diff
@@ -19,7 +19,7 @@ description: "Learn how your organization can export a specific user's content f
 
 This article describes how a Microsoft 365 tenant admin can export a specific user’s data from Project for the web. The admin can then choose to view the user’s data and decide what data they want to make available to the user. 
 
-Project for the web data is stored in the [Common Data Service (CDS)](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro) in Microsoft PowerApps. This article describes how you can:  
+Project for the web data is stored in the [Dataverse](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro) in Microsoft PowerApps. This article describes how you can:
```
Comment (JohanFreelancer9) [inline ProjectWeb/export-user-data-from-project-for-the-web.md:22]:
```suggestion
Project for the web data is stored in [Dataverse](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro) in Microsoft PowerApps. This article describes how you can:  
```

File: ProjectWeb/export-user-data-from-project-for-the-web.md [hunk 12]
```diff
@@ -41,16 +41,16 @@ If you also want to export and view information on specific projects:
 
 Depending on the type of user data you need to find, there are two paths you can take in searching for and exporting your user's data.
 
-- **Find data about your users project and roadmap objects in CDS** - Use the Advanced Find feature in the Dynamics 365 Admin Center to find all the users data that is contained in CDS (for example, objects related to their projects and roadmaps).
+- **Find data about your users project and roadmap objects in CDS** - Use the Advanced Find feature in the Dynamics 365 Admin Center to find all the users data that is contained in Dataverse (for example, objects related to their projects and roadmaps).
```
Comment (JohanFreelancer9) [inline ProjectWeb/export-user-data-from-project-for-the-web.md:44]:
```suggestion
- **Find data about your users project and roadmap objects in Dataverse** - Use the Advanced Find feature in the Dynamics 365 Admin Center to find all the users data that is contained in Dataverse (for example, objects related to their projects and roadmaps).
```

File: ProjectWeb/export-user-data-from-project-for-the-web.md [hunk 13]
```diff
@@ -41,16 +41,16 @@ If you also want to export and view information on specific projects:
...
+## Find user data in Dataverse with the Advanced Find search feature 
 
-Project for the web user information that resides in CDS - such as roadmap and project objects and properties - are located in specific Dynamics 365 solutions.  The Advanced Find search feature in the Dynamics 365 admin center can query across entities in these solutions to find the information you need.
+Project for the web user information that resides in Dataverse - such as roadmap and project objects and properties - are located in specific Dynamics 365 solutions.  The Advanced Find search feature in the Dynamics 365 admin center can query across entities in these solutions to find the information you need.
 
-### Understand Project for the web CDS data and where it resides
+### Understand Project for the web Dataverse data and where it resides
 
-When looking for a specific users Project for the web data in Dynamics 365 CDS, it is located in these five Dynamics 365 solutions:
+When looking for a specific users Project for the web data in Dynamics 365 Dataverse, it is located in these five Dynamics 365 solutions:
```
Comment (JohanFreelancer9) [inline ProjectWeb/export-user-data-from-project-for-the-web.md:53]:
```suggestion
When looking for a specific user's Project for the web data in Dynamics 365 Dataverse, it is located in these five Dynamics 365 solutions:
```

File: ProjectWeb/export-user-data-from-project-for-the-web.md [hunk 14]
```diff
@@ -121,7 +121,7 @@ Assuming you've used Advanced Find search to query for the user's projects and h
 
 After you unhide the columns in the spreadsheet, looks for the name of the project, and then look for the corresponding value in the Project column to find the Project ID for the project.
```
Comment (JohanFreelancer9) [inline ProjectWeb/export-user-data-from-project-for-the-web.md:122]:
```suggestion
After you unhide the columns in the spreadsheet, look for the name of the project, and then look for the corresponding value in the Project column to find the Project ID for the project.
```

File: ProjectWeb/handling-data-for-project-for-the-web-and-roadmap.md [hunk 15]
```diff
@@ -11,7 +11,7 @@ search.appverid:
 - PJO150
 - MET150
 localization_priority: Normal
-description: "Learn how Project for the web uses the Common Data Service to store and manage data."
+description: "Learn how Project for the web uses the Dataverse to store and manage data."
```
Comment (JohanFreelancer9) [inline ProjectWeb/handling-data-for-project-for-the-web-and-roadmap.md:14]:
```suggestion
description: "Learn how Project for the web uses the Dataverse to store and manage data."
```
```suggestion
description: "Learn how Project for the web uses Dataverse to store and manage data."
```

File: ProjectWeb/handling-data-for-project-for-the-web-and-roadmap.md [hunk 16]
```diff
@@ -20,7 +20,7 @@ This article gives you an overview of how data is handled for two Project servic
 
 Project Online is built on SharePoint Online and its data is stored in Office 365 (specifically in SharePoint Online).
 
-Project for the web is built on the [Microsoft Power Platform](https://powerplatform.microsoft.com/). Project for the web data is stored in the [Common Data Service (CDS)](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro).
+Project for the web is built on the [Microsoft Power Platform](https://powerplatform.microsoft.com/). Project for the web data is stored in the [Dataverse](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro).
```
Comment (JohanFreelancer9) [inline ProjectWeb/handling-data-for-project-for-the-web-and-roadmap.md:23]:
```suggestion
Project for the web is built on the [Microsoft Power Platform](https://powerplatform.microsoft.com/). Project for the web data is stored in [Dataverse](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro).
```

File: ProjectWeb/handling-data-for-project-for-the-web-and-roadmap.md [hunk 17]
```diff
@@ -20,7 +20,7 @@ This article gives you an overview of how data is handled for two Project servic
...
 
 
 > [!Note]
```
Comment (JohanFreelancer9) [inline ProjectWeb/handling-data-for-project-for-the-web-and-roadmap.md:26]:
```suggestion
> [!NOTE] 
```

File: ProjectWeb/handling-data-for-project-for-the-web-and-roadmap.md [hunk 18]
```diff
@@ -37,13 +37,13 @@ You can remove all Project for the web or Roadmap data by [removing the entire s
 
 You can look for the location of where your Project for the web and Project Online data is stored by seeing [Explore where Office 365 stores your customer data](https://products.office.com/where-is-your-data-located?rtc=1). 
 
-### Project for the web and the Common Data Service
+### Project for the web and the Dataverse
```
Comment (JohanFreelancer9) [inline ProjectWeb/handling-data-for-project-for-the-web-and-roadmap.md:40]:
```suggestion
### Project for the web and Dataverse
```

File: ProjectWeb/project-architecture-overview.md [hunk 19]
```diff
@@ -52,7 +52,6 @@ The key Project apps described in this article are available in the following Pr
 |Project Online Desktop Client<br/> |  Not Available<br/> |Available | Available|
 
 
-
 > [!Note]
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-architecture-overview.md:55]:
```suggestion
> [!NOTE] 
```

File: ProjectWeb/project-architecture-overview.md [hunk 20]
```diff
@@ -62,14 +61,14 @@ For more details on Project Plans, see the [Microsoft Project Service Descriptio
...
+Project for the web is built on the [Microsoft Power Platform](https://powerplatform.microsoft.com). The Power Platform consists of [PowerApps](https://docs.microsoft.com/powerapps/powerapps-overview), [Power Automate](https://docs.microsoft.com/power-automate/getting-started), [Power BI](https://docs.microsoft.com/power-bi/fundamentals/power-bi-overview), and the [Dataverse](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro). Integration with the Microsoft Power Platform lets you easily leverage its components to [create custom business solutions](https://developer.microsoft.com/project/blogs/developing-applications-and-reports-using-the-new-project/) and do advanced analytics and reporting on project data.
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-architecture-overview.md:64]:
```suggestion
Project for the web is built on the [Microsoft Power Platform](https://powerplatform.microsoft.com). The Power Platform consists of [PowerApps](https://docs.microsoft.com/powerapps/powerapps-overview), [Power Automate](https://docs.microsoft.com/power-automate/getting-started), [Power BI](https://docs.microsoft.com/power-bi/fundamentals/power-bi-overview), and [Dataverse](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro). Integration with the Microsoft Power Platform lets you easily leverage its components to [create custom business solutions](https://developer.microsoft.com/project/blogs/developing-applications-and-reports-using-the-new-project/) and do advanced analytics and reporting on project data.
```

File: ProjectWeb/project-architecture-overview.md [hunk 21]
```diff
@@ -62,14 +61,14 @@ For more details on Project Plans, see the [Microsoft Project Service Descriptio
...
 
 Just like Project Online, users can access their Project for the web projects through the [Project Home page](https://support.office.com/article/get-started-with-project-home-a3b38418-35e7-4df4-8e4a-ba6a4fa0562a).  It will by default list projects that were recently viewed, owned by, or shared with the user.
 
 ### Data storage
-Project for the web data is saved to the Common Data Service (CDS). CDS is part of the Microsoft Power Platform, which Project for the web is built on.  
+Project for the web data is saved in the Dataverse. The Dataverse is part of the Microsoft Power Platform, which Project for the web is built on.
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-architecture-overview.md:69]:
```suggestion
Project for the web data is saved in Dataverse. Dataverse is part of the Microsoft Power Platform, which Project for the web is built on.  
```

File: ProjectWeb/project-for-the-web-get-started-guide-for-admins.md [hunk 22]
```diff
@@ -54,16 +55,14 @@ Project for the web is designed to be easy-to-use and intuitive. You can use the
...
 
 [Connect to Project data through Power BI Desktop](connect-to-project-for-the-web-data-through-powerbi-desktop.md)
 
-## Common Data Service
-Your Project for the web and Roadmap data is stored in your [Common Data Service](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro) default instance. Initial use of Project for the web or Roadmap in your tenant will automatically create the default CDS instance. 
-
-> [!NOTE]
-> Project for the web is only supported on your default CDS instance.
+## Dataverse
+Your Project for the web and Roadmap data is stored in the [Dataverse (formerly Common Data Service)](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro).
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-for-the-web-get-started-guide-for-admins.md:63]:
```suggestion
Your Project for the web and Roadmap data is stored in [Dataverse (formerly Common Data Service)](https://docs.microsoft.com/powerapps/maker/common-data-service/data-platform-intro). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**